src-worker/external/targetsPool.ts

```diff
@@ -63,17 +63,16 @@ export class TargetsPool {
     }
 
     // Load proxyes from each resource
-    let loadedTargetsList = [] as Target[]
+    const loadedTargetsList = [] as Target[]
     for (const url of this.sources) {
       const targets = await this.loadFromURL(url)
       if (typeof targets === 'string') {
         console.warn(targets)
       } else {
-        loadedTargetsList = [...loadedTargetsList, ...targets]
+        loadedTargetsList.push(...targets.filter((t) => t.method === 'get' || t.method === 'post'))
```

Comment:
new target types where added. please check 'udp_flood' and 'slowloris'

Reply:
let me think how to deal with it